assets/asset_manager.h

```diff
@@ -25,6 +25,34 @@ class AssetManager final : public AssetResolver {
 
   void PushBack(std::unique_ptr<AssetResolver> resolver);
 
+  //--------------------------------------------------------------------------
+  /// @brief      Replaces an asset resolver of the specified `type` with
+  ///             `updated_asset_resolver`. The matching AssetResolver is
+  ///             removed and replaced with `updated_asset_resolvers`.
+  ///
+  ///             AssetResolvers should be updated when the exisitng resolver
```

Comment:
typo: "existing"

(This also appears in the comments for UpdateAssetResolverByType in other classes)